docs/_data/controllers/MFKZQ01LM.yml

```
@@ -0,0 +1,16 @@
+name: MFKZQ01LM Xiaomi Smart Cube (Aqara)
+device_support: 
+  - Light (E1810Controller; 350ms delay)
+integrations:
```

Since there is no natural way of using this controller, I would add a note explaining how the controller works and in the future and that its behaviour can be changed with custom controller. Once I merge and change the shake action to minimum brightness, I will then add also the behaviour of this controller.
```suggestion
note: >-
  This controller does not have a self-explained way to use it since its main purpose is to customize it, this is why we recommend to customize it with <a href="/controllerx/how-to/custom-controllers">Custom controllers</a>. However, we gave a default behaviour to this controller.
integrations:
```